app.json

```diff
@@ -1,11 +1,10 @@
 {
   "expo": {
     "name": "my-project",
-    "description":"my first todo app",
+    "description": "my first todo app",
     "slug": "my-project",
     "version": "1.0.0",
     "orientation": "portrait",
-    "sdkVersion": "36.0.0",
```

Comment:
you removed sdk version?
just need to update it

Reply:
mam it had been updted and removed directly by the exxpo itself

Reply:
ok